common-kafka/src/test/java/com/cerner/common/kafka/consumer/ProcessingPartitionTest.java

```diff
@@ -552,12 +553,13 @@ public void getLastCommittedOffset_committedOffsetAfterLatest() {
         // Since our offset is out of range we will fall back to the offset based on our reset strategy
         // We set the reset strategy to earliest so this should give earliest offset
         assertThat(partition.getLastCommittedOffset(), is(25L));
-        verify(consumer).commitSync(Collections.singletonMap(topicPartition, any(OffsetAndMetadata.class)));
+        verify(consumer).commitSync(anyMap());
```

Comment:
Is there any reason why this was changed to anyMap()?

Reply:
The new matchers work a bit differently and the tests were failing even though the commitSync method was being called with the same arguments, so this was my solution to get them passing.  I got them working with more specificity with:
https://github.com/cerner/common-kafka/pull/51/commits/ccadd2724b228a449b7c30db385cb315cfba463a